README.md

```diff
@@ -21,6 +21,10 @@ You can then view your metrics at: http://0.0.0.0:9394/metrics
 For extra Puma metrics, add this to `config/puma.rb`:
 
 ```ruby
+before_fork do
+  Rails.application.config.before_fork_callbacks.each(&:call)
+end
```

Comment:
This isn't for extra Puma metrics, this is so the server starts.  It should be in a new paragraph above line 21.

Reply:
yeah I didn't see the extra metric line. Moving to the main setup block